superset/tasks/utils.py

```diff
@@ -133,7 +133,7 @@ def fetch_csrf_token(
             data = json.loads(body)
             res = {"X-CSRF-Token": data["result"]}
             if session_cookie is not None:
-                res["Cookie"] = session_cookie
+                res["Cookie"] = f"{session_cookie_name}={session_cookie}"
```

Comment:
Shallow comment here since I have limited time today but still want to provide feedback/guidance, but are we sure that doesn't break anything for anyone out there?

Reply:
Thanks for running the extra tests. I'm just wondering how it works now for some people without specifying the cookie name as you do here. If it works now for them, how would changing what works with a name prefix not break things (?)

Reply:
Ok, I did a bit more research and got GPT's input here https://chatgpt.com/share/6752157e-8dc0-8010-b7c8-afcd7e270512 .

My recommendation, to be cautious, is to add a line in `UPDATING.md` as the last item in the `Next` section that says something like 

- [31173](https://github.com/apache/superset/pull/30099) NOTE: modified `fetch_csrf_token` to align with HTTP standards, modifying how cookies are handled. If you hit issues related to CSRF, please report as an issue while referring to this PR

Reply:
I agree that there may be edge cases or environments where this adjustment could impact behavior. I will add a note to `UPDATING.md` as you suggested, explaining the change and advising users to report any CSRF-related issues while referencing PR #31173